soccer/radio/PacketConvert.cpp

```diff
@@ -72,6 +72,57 @@ Packet::RadioRx convert_rx_rtp_to_proto(const rtp::RobotStatusMessage& msg) {
     return packet;
 }
 
+void construct_tx_proto(Packet::RadioTx& radioTx,
+                        const std::array<RobotIntent, Num_Shells>& intents) {
+    // I'm assuming this is necessary to do logging. idk
```

Comment:
This constructs the `RadioTx` from the intents, right?

Reply:
yep